components/operator/Earthfile

```diff
@@ -67,17 +67,14 @@ generate:
 helm-update:
     FROM core+builder-image
     RUN curl -s https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh | bash -s -- 4.5.4 /bin
-
     WORKDIR /src
     COPY --pass-args (+manifests/config) /src/config
     COPY --dir helm hack .
-
+    RUN rm -f helm/templates/crd/*
```

Comment:
Why move this code BEFORE the kustomize build?
The file will not be removed at the end.